README.adoc

```diff
@@ -357,26 +357,22 @@ might be fully or partially complete. If the document is fully complete, then yo
 annotation scanning entirely by setting the `mp.openapi.scan.disable` MicroProfile Config property to `true`.
 If the document is partially complete, then you can augment it with code.
 
-You can find a complete OpenAPI document in the [hotspot file=0]`src/main/webapp/META-INF/openapi.yaml` file.
-This document is the same as your current OpenAPI document with additional APIs for the `/inventory/properties`
-endpoint. To make use of this document, open this document in your favorite text editor and uncomment
-all of its lines. 
+To use the pregenerated OpenAPI document, create the OpenAPI document yaml file.
```

Comment:
@tt-le Thanks for making updates! Could we change "pregenerated" to "pre-generated"? Also "yaml" to "YAML"?

Reply:
@Charlotte-Holt Updated!